src/activity/events/components/EventPagination.js

```diff
@@ -0,0 +1,59 @@
+import { useState, useEffect } from "react";
+import PropTypes from "prop-types";
+import { Box, Pagination } from "@mui/material";
+
+function EventPagination({ onPageChanged, page, pageSize, total }) {
+	const [paginationSize, setPaginationSize] = useState("large");
+	const [screenSize, getDimension] = useState({
+		dynamicWidth: window.innerWidth,
+	});
+
+	const setDimension = () => {
+		getDimension({
+			dynamicWidth: window.innerWidth,
+		});
+	};
+
+	useEffect(() => {
+		window.addEventListener("resize", setDimension);
+		if (screenSize.dynamicWidth > 960) {
+			setPaginationSize("large");
+		} else if (screenSize.dynamicWidth > 640) {
+			setPaginationSize("medium");
+		} else {
+			setPaginationSize("small");
+		}
+		return () => {
+			window.removeEventListener("resize", setDimension);
+		};
+	}, [screenSize]);
+
+	return (
+		<Box className="pagination-box">
```

Comment:
When you get to the middle of the pages, you can only click to the next number (5 to 6, but can't click 5 to 7)
<img width="544" alt="Screen Shot 2022-08-24 at 4 09 31 PM" src="https://user-images.githubusercontent.com/36748732/186540099-7af78719-f4f1-4292-b53a-4278744e4b4b.png">

It would be nice to skip ahead like in the earlier pages, where I could click from 3 to 5.
<img width="532" alt="Screen Shot 2022-08-24 at 4 19 31 PM" src="https://user-images.githubusercontent.com/36748732/186540217-7be409fa-9cf5-4783-90ce-a3f49505351b.png">

Reply:
Resolved

![sibling-count](https://user-images.githubusercontent.com/78233689/186576686-0a0cd2c5-6a82-4337-8509-cd6a20815a34.png)